app/views/shared/_footer.html.slim

```diff
@@ -0,0 +1,9 @@
+.footer
+  .nav.navbar.navbar-expand-lg.navbar-light.bg-light
+    ul.navbar-nav
+      li.nav-item.nav-link
+        = link_to 'Social Link'
+      li.nav-item.nav-link
+        = link_to 'Social Link'
+      li.nav-item.nav-link
+        = link_to 'Social Link'
```

Comment:
Add links to static pages (About, Contacts, etc..) to the footer, and probably remove them from `pages/index` until we fix the wireframes for the main layout